src/inventory_ui.cpp

```diff
@@ -17,37 +17,110 @@
 #include "itype.h"
 
 #include <string>
+#include <tuple>
 #include <vector>
 #include <map>
 #include <sstream>
 #include <algorithm>
 
-bool inventory_entry::operator == ( const inventory_entry &other ) const {
+static const item_category weapon_held_cat( "WEAPON HELD", _( "WEAPON HELD" ), -200 );
```

Comment:
Could you rephrase your question? I have found neither in the code (except of the `backpack` which is a clothing item).